[2021-11-27 11:32:09,074] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: elt.upload scheduled__2021-11-27T04:31:00+00:00 [queued]>
[2021-11-27 11:32:09,080] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: elt.upload scheduled__2021-11-27T04:31:00+00:00 [queued]>
[2021-11-27 11:32:09,081] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 11:32:09,081] {taskinstance.py:1242} INFO - Starting attempt 1 of 2
[2021-11-27 11:32:09,081] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-11-27 11:32:09,089] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): upload> on 2021-11-27 04:31:00+00:00
[2021-11-27 11:32:09,091] {standard_task_runner.py:52} INFO - Started process 10907 to run task
[2021-11-27 11:32:09,098] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'elt', 'upload', 'scheduled__2021-11-27T04:31:00+00:00', '--job-id', '544', '--raw', '--subdir', 'DAGS_FOLDER/processing.py', '--cfg-path', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmpko3ra7vf', '--error-file', '/var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T/tmpnybbpryf']
[2021-11-27 11:32:09,100] {standard_task_runner.py:77} INFO - Job 544: Subtask upload
[2021-11-27 11:32:09,133] {logging_mixin.py:109} INFO - Running <TaskInstance: elt.upload scheduled__2021-11-27T04:31:00+00:00 [running]> on host MacBookPro.local
[2021-11-27 11:32:09,160] {taskinstance.py:1427} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=elt
AIRFLOW_CTX_TASK_ID=upload
AIRFLOW_CTX_EXECUTION_DATE=2021-11-27T04:31:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-11-27T04:31:00+00:00
[2021-11-27 11:32:09,161] {subprocess.py:62} INFO - Tmp dir root location: 
 /var/folders/hj/g8v71bv11_gb814qbbwch8zc0000gn/T
[2021-11-27 11:32:09,162] {subprocess.py:74} INFO - Running command: ['bash', '-c', '/Users/Home/Airflow/tasks/cleaning.py /Users/Home/Airflow/output worldbank']
[2021-11-27 11:32:09,168] {subprocess.py:85} INFO - Output:
[2021-11-27 11:32:09,217] {subprocess.py:89} INFO - rm: /Users/Home/Airflow/output/*API_DT.ODA.ODAT.GI.ZS_DS2_en_csv_v2_3164289.csv: No such file or directory
[2021-11-27 11:32:09,218] {subprocess.py:89} INFO - API_DT.ODA.ODAT.GI.ZS_DS2_en_csv_v2_3164289.csv
[2021-11-27 11:32:09,223] {subprocess.py:93} INFO - Command exited with return code 0
[2021-11-27 11:32:09,238] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=elt, task_id=upload, execution_date=20211127T043100, start_date=20211127T043209, end_date=20211127T043209
[2021-11-27 11:32:09,275] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-11-27 11:32:09,289] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
